src/main/php/web/frontend/Templates.class.php

```diff
@@ -8,6 +8,7 @@ interface Templates {
    * @param  string $name Template name
    * @param  [:var] $context
    * @param  io.streams.OutputStream $out
```

Comment:
The `$fragment` parameter is **not** part of this interface to prevent this:

*PHP Fatal error:  Declaration of web\frontend\Templates@1::write($template, $context, $out) must be compatible with web\frontend\Templates::write($name, $context, $out, $fragment = null)*